### DISCUSSION TITLE
MySQL NDBCluster

### DISCUSSION THREAD

**Discussion by shaun-creakyhat** [sqlalchemy/sqlalchemy#11927] [Usage Questions]
we have been using sqlalchemy with a docker container but are moving to using it with a MySQL cluster.

when creating tables it now need to set the engine to NDBCLUSTER

eg:

Create Table TESING (
  UIUID char(36),
  NAME char(16)
) Engine=NDBCLUSTER;

for the docker mysql it does

Create Table TESING (
  UIUID char(36),
  NAME char(16)
) Engine=InnoDB;

is this possible I don't see anything in the documentation

**Comment by CaselIT** [comment]
Hi

It's documented here https://docs.sqlalchemy.org/en/20/dialects/mysql.html#create-table-arguments-including-storage-engines
